tests/OK/inductive_type.lp

```diff
@@ -33,7 +33,7 @@ set builtin "eqind" ≔ eq_ind
 /////////////////////////////
 
 ////////////////// Primary color
-
+set debug +g
```

Comment:
Please avoid committing debug flags (or at least remember to remove them before merging)

Reply:
Yes! There is the same problem in the directory `KO`.